packages/flutter_tools/lib/src/version.dart

```diff
@@ -15,6 +15,21 @@ import 'cache.dart';
 import 'convert.dart';
 import 'globals.dart' as globals;
 
+/// The flutter GitHub repository.
+String get kFlutterGit => globals.platform.environment['FLUTTER_GIT_URL'] ?? 'https://github.com/flutter/flutter.git';
```

Comment:
`k` is really only for constants. This shouldn't be top-level, since it needs to inject Platform